compose_files/docker-compose-forger.yml

```diff
@@ -16,12 +17,13 @@ networks:
 
 services:
   zend:
-    image: zencash/zen-node:${ZEN_TAG}
-    container_name: zend
+    image: "zencash/zen-node:${ZEND_TAG}"
+    container_name: "${ZEND_CONTAINER_NAME:-zend}"
     restart: always
+    stop_grace_period: 10m
     networks:
       evmapp_network:
-        ipv4_address: 10.10.50.10
+        ipv4_address: ${ZEND_IP_ADDRESS}
```

Comment:
I think you should try use double quotes if works as expected here as well to follow your pattern before(all bash env vars enclosed in double quotes)